.github/workflows/build.yml

```diff
@@ -24,7 +24,8 @@ jobs:
       fail-fast: false
 
       matrix:
-        os: [ 'windows-latest', 'ubuntu-latest', 'macos-latest' ]
+        # The second ubuntu-latest agent is used to build linux-musl-x64
+        os: [ 'windows-latest', 'ubuntu-latest', ubuntu-latest', 'macos-latest' ]
```

Comment:
This will not work, see discussion here: https://github.community/t/create-matrix-with-multiple-os-and-env-for-each-one/16895/6

Reply:
Thanks for the link. I'll update it to use include.